src/Scanner.js

```diff
@@ -223,26 +220,16 @@ class DefaultScanner {
         return this.mode === 'stopped';
     }
 
-    /**
-     * Gets the forced form only for the first call, subsequent calls return null
-     * @returns {HTMLElement|null}
-     */
-    getAndClearForcedForm() {
-        if (this._hasUsedForcedForm) return null;
-
-        if (this._forcedForm === null) {
-            this._forcedForm = this.device.settings.siteSpecificFeature?.getForcedForm() || null;
-        }
-
-        this._hasUsedForcedForm = true;
-        return this._forcedForm;
-    }
-
     /**
      * @param {HTMLElement} input
      * @returns {HTMLElement}
      */
     getParentForm(input) {
+        this._forcedForm = this.device.settings.siteSpecificFeature?.getForcedForm() || null;
+        if (this._forcedForm?.contains(input)) {
+            return this._forcedForm
+        }
+
```

Comment:
It seems to me that moving the checks inside `getParentForm` is the most semantically correct way to do it. The point of these forced forms is that we don't need to get a parent form, because it's enforced via config. I have also removed the "check only once" logic because forms come and go. If we check only at the start we might miss a forced form that comes later on in the page lifecycle. Should we add a test for that case? Meaning:

- Page loads, forced form is not in the DOM
- Click a button, forced form appears
- Expect forced form to be scanned correctly